index.md

```diff
@@ -13,7 +13,7 @@ Probo is both a SaaS app, available at app.probo.ci, as well as an open source s
 ## Getting Started
 The first step to using Probo is logging in to the Probo app, where you can sync your repositories, view your builds, and access build information like the console output.
 
-Go to the [Probo app](https://app.probo.ci/ "Probo app") and log in with your **GitHub** or **Bitbucket** account. On the authorization page, grant Probo permission to access your repos. Back in the Probo app, enter your user information and click Submit.
+Go to the [Probo app](https://app.probo.ci/ "Probo app") and log in with your **GitHub**, **Bitbucket**, or **Gitlab** account. On the authorization page, grant Probo permission to access your repos. Back in the Probo app, enter your user information and click Submit.
```

Comment:
Can you capitalize the L in GitLab?

Reply:
Woops, missed that one! It autocorrected me in Sublime...